src/calendarUtils.ts

```diff
@@ -80,6 +80,7 @@ export interface MonthViewDay extends WeekDay {
 export interface MonthView {
   rowOffsets: number[];
   days: MonthViewDay[];
+  daysInWeek: number;
```

Comment:
Can you rename this to `totalDaysVisibleInWeek` to make it a bit more verbose. At first glance I thought it was an array of visible days rather than just the count.